config.py

```diff
@@ -65,4 +65,17 @@ def configure(advanced):
                     private=True))
 
 
+conf.registerGroup(Fedora, 'karma')
+conf.registerGlobalValue(
+    Fedora.karma, 'db_path',
+    registry.String('/var/tmp/supybot-karma.db',
+                    """Path to a karma db on disk"""))
+conf.registerGlobalValue(
+    Fedora.karma, 'unaddressed',
+    registry.Boolean(True, "Allow unaddressed karma commands"))
```

Comment:
I am not quite sure to understand what this option means, could we expand on it somewhere?